osu.Game/Screens/Menu/MainMenuButton.cs

```diff
@@ -156,6 +157,15 @@ protected override void OnNewBeat(int beatIndex, TimingControlPoint timingPoint,
             rightward = !rightward;
         }
 
+        public bool OnNavigation()
+        {
+            return this.OnHover(new HoverEvent(new InputState()));
+        }
+
+        public void OnNavigationLost()
+        {
+            this.OnHoverLost(new HoverLostEvent(new InputState()));
+        }
```

Comment:
This is not the right way of marking a button as hovered. Have a look at `VolumeOverlay` and `SelectionCycleFillFlowContainer`, which provide something similar to what you're trying to achieve.